app/services/idv/steps/verify_step.rb

```diff
@@ -14,6 +14,23 @@ def enqueue_job
 
         pii_from_doc[:uuid_prefix] = ServiceProvider.find_by(issuer: sp_session[:issuer])&.app_id
 
+        if pii_from_doc[:ssn].present?
+          throttle = Throttle.for(
+            target: Pii::Fingerprinter.fingerprint(pii_from_doc[:ssn]),
+            throttle_type: :proof_ssn,
+          )
+
+          if throttle.throttled_else_increment?
+            @flow.analytics.track_event(
+              Analytics::THROTTLER_RATE_LIMIT_TRIGGERED,
+              throttle_type: :proof_ssn,
+              step_name: self.class,
+            )
+            redirect_to idv_session_errors_failure_url
```

Comment:
This is one of the screens where we use the [JavaScript spinner submission](https://github.com/18F/identity-idp/blob/main/app/javascript/packs/form-steps-wait.jsx), yeah? I _think_ it might handle these redirects gracefully, but worth double-checking, since normally it's dealing with happy-path redirects:

https://github.com/18F/identity-idp/blob/569959659a2ab3a259c27b82511d2c7c2cca6252/app/javascript/packs/form-steps-wait.jsx#L113

Reply:
Confirmed it worked as expected! The error message is a little funky, but that may be OK for now

<img width="668" alt="Screen Shot 2021-08-19 at 9 21 44 AM" src="https://user-images.githubusercontent.com/458784/130106025-3e103644-d079-4b56-beca-aadf093ea3d9.png">

How important do you think it is to have a feature test that runs through this with JS enabled?

Reply:
> How important do you think it is to have a feature test that runs through this with JS enabled?

Could be nice, but not a blocker. Do we at least have one for the happy path?

Reply:
Yup! Happy path covered by https://github.com/18F/identity-idp/blob/main/spec/features/idv/doc_auth/verify_step_spec.rb